### DISCUSSION TITLE
manager/configuration-and-advanced-settings/How-to-backup-and-restore-panel-on-Hiddify/

### DISCUSSION THREAD

**Discussion by giscus** [hiddify/Hiddify-Manager#4658] [Content Help کمک به اصلاح محتوا]
# manager/configuration-and-advanced-settings/How-to-backup-and-restore-panel-on-Hiddify/

The proxy manager platform.

https://hiddify.com/manager/configuration-and-advanced-settings/How-to-backup-and-restore-panel-on-Hiddify/

**Comment by web2best** [comment]
How can I re-enter the same proxy path into a new panel - a duplicate of the old one on the old server? It gives an error that this proxy path is already in use.
